.github/workflows/release-deploy-all.yml

```diff
@@ -101,8 +100,6 @@ jobs:
         run: |
           echo Git Environment: ${{ inputs.environment }}
           echo Git Red: ${{ inputs.gitRef }}
-          echo BUILD NAMESPACE: $BUILD_NAMESPACE
```

Comment:
Why did we remove the echo of the build_namespace ?

Reply:
its also not printing so removed.

<img width="790" alt="image" src="https://github.com/bcgov/SIMS/assets/62901416/68a1d76b-d6d4-460e-8c8d-b056ef0b249f">

Reply:
Correct me if wrong. I see the echo happening just under that. 
![image](https://github.com/bcgov/SIMS/assets/54600590/28a0fce5-d3c6-4fa8-96ff-d53bcfe61b0a)

Reply:
My bad missed it in the printing below, reverted the change back :)